packages/rum-core/src/performance-monitoring/metrics.js

```diff
@@ -317,12 +317,9 @@ export class PerfEntryRecorder {
   }
 
   start(type) {
-    /**
-     * Safari throws an error when PerformanceObserver is
-     * observed for unknown entry types as longtasks and lcp is
-     * not supported at the moment
-     */
-    try {
+    const supportedEntryTypes =
+      window.PerformanceObserver?.supportedEntryTypes || []
```

Comment:
We have the [isPerfTypeSupported](https://github.com/elastic/apm-agent-rum-js/blob/main/packages/rum-core/src/common/utils.js#L396) util